### PR TITLE
Fix compatibility with `vercel/ncc`

### PR DESCRIPTION
**Pull request by Idered**
related #67

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const normalizePackageData = require('normalize-package-data');
 
 // Prevent caching of this module so module.parent is always accurate
 delete require.cache[__filename];
-const parentDir = path.dirname(module.parent.filename);
+const parentDir = path.dirname(module.parent?.filename ?? '.');
```
Comment (sindresorhus) [inline index.js:15]: This syntax is not supported in the targeted Node.js version.

Comment (Idered) [reply index.js]: Sorry, that was quick patch made in browser. Fixed and tested locally.

**Comment by Idered** [comment]
Ok, looks like it's failing on latest node. I'll take a look later

**Comment by Idered** [comment]
@sindresorhus fixed 👌

**Pull request merged**